packages/web/components/Guild/GuildForm.tsx

```diff
@@ -180,7 +185,9 @@ export const GuildForm: React.FC<Props> = ({
             isInvalid={!!errors.guildname}
             background="dark"
           />
-          <span>A unique identifier for your guild, like a username.</span>
+          <span className="description">
```

Comment:
I would use `chakra.span` or `<Text as="span">`, remove the class, and move the `Box`'s `sx` attribute down here.

Though, honestly, given how the elements are used on the page, you could use a `block` element, so just `<Text/>`.